templates/NamedOperationGroupFile.stencil

```diff
@@ -23,8 +23,8 @@ public final class {{ group.name }} {
         self.commonOptions = client.commonOptions
     }
 
-    public func url(forTemplate templateIn: String, withKwargs kwargs: [String: String]? = nil, and addedParams: [QueryParameter]? = nil) -> URL? {
-        return client.url(forTemplate: templateIn, withKwargs: kwargs, and: addedParams)
+    public func url(forTemplate templateIn: String, withHost hostIn: String? = nil, withKwargs kwargs: [String: String]? = nil, and addedParams: [QueryParameter]? = nil) -> URL? {
+        return client.url(forTemplate: templateIn, withHost: hostIn, withKwargs: kwargs, and: addedParams)
```

Comment:
This also looks like something that should be in AzureCore.

Reply:
Open issue https://github.com/Azure/azure-sdk-for-ios/issues/534 for AzureCore